Add italian language

Bl5ckj5ck opened the pull request:
Hey guys! Hi @serhack ! 
My passion for Monero started thanks to Mastering Monero and I find this resource incredibly interesting. Since it's very helpful, I'd like to add the Italian language so that the book can attract also those who prefer to read it in their native language. I've already completed the translation of the first chapter (1.md) and I opened a pull request so you can see the changes and approve them if you think that it is up to the quality standard or otherwise reject them. 
If you approve the changes, then i'd like to go ahead with the translation of the whole book, so please don't merge the changes until i complete it.  Eventually I'd like to create a CCS Proposal for the remaining chapters since this takes much time. Please let me know what you think about!

Bl5ckj5ck committed:
Create new folder translations/it and copy all files to be translated

Bl5ckj5ck committed:
Add translation 1.md

Bl5ckj5ck committed:
Update1 1.md it

I made few typo corrections and changed the syntax of some sentences in order to improve their flow

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/monerobook/monerobook?pullRequest=109) <br/>All committers have signed the CLA.

Bl5ckj5ck committed:
Add it-translation 2.md

Bl5ckj5ck committed:
Update 2.md it

I made few typo corrections and changed the syntax of some sentences in order to improve their flow

Bl5ckj5ck committed:
Add it-translation 3.md
